BackEndApp/authv1/views.py

```diff
@@ -93,3 +96,96 @@ def logout(request):
         status = 500
 
     return JsonResponse({"message": message}, status=status)
+
+
+@api_view(["POST"])
+def forgot_password(request):
+    try:
+        username = request.data.get("username")
+        user = User(username=username, password=None)
+        this_user = user.find()
+
+        if user is None:
+            message = "User does not exist"
+            status = 401
+
+        otp = OTP(this_user)
+        generated_otp = otp.create()
+
+        user_email = this_user["email"]
+        email = EmailTemplates(this_user)
+        subject, msg = email.forgot_password(username, generated_otp)
+        send_mail(subject, msg, EMAIL_HOST_USER, [user_email])
+
+        message = "Email sent successfully."
+        status = 200
+
+    except Exception as e:
+        message = "Some error occurred!! Please try again."
+        status = 500
+
+    return JsonResponse({"message": message}, status=status)
+
+
+@api_view(["POST"])
+def verify_otp(request):
+    try:
+        username = request.data.get("username")
+        received_otp = request.data.get("received_otp")
+        user = User(username=username, password=None)
+        user = user.find()
+
+        otp = OTP(user)
+        otp_verified = otp.verify(received_otp)
+
```

Comment:
Need to check if the email is verified, before giving a success message

Reply:
fixing this.